docs/ui/auth/fragments/web/authenticator.md

```diff
@@ -83,6 +83,36 @@ _App.vue_
 
 </docs-filter>
 
+<amplify-callout warning>
+
+To enable password manager, you need to wrap the Authenticator component with `amplify-auth-container` component.
```

Comment:
```suggestion
To enable password manager, you need to wrap the Authenticator component with an `amplify-auth-container` component.
```
😄

Reply:
Actually wait, it's `<AmplifyAuthContainer>` for React right?  Can we split this into React and non-React as well?

Reply:
Right, will do.